package.json

```diff
@@ -48,7 +48,7 @@
     "electron-releases": "^3.86.0",
     "eslint": "^5.8.0",
     "eslint-config-standard": "^12.0.0",
-    "eslint-plugin-import": "^2.14.0",
+    "eslint-plugin-import": "^2.16.0",
```

Comment:
So this was installed but not used before?

A very good plugin to have nonetheless.

Reply:
It was used via [standard](https://github.com/standard/eslint-config-standard) which I removed in preparation of prettier since they both care so much about formatting of the code :)